src/wasm/wat-parser.cpp

```diff
@@ -1881,6 +1881,7 @@ struct ParseDefsCtx : TypeParserCtx<ParseDefsCtx> {
   Result<> makeStructSet(Index pos, HeapType type, Index field) {
     assert(type.isStruct());
     const auto& fields = type.getStruct().fields;
+    WASM_UNUSED(fields);
     assert(fields.size() > field);
```

Comment:
Since `fields` isn't otherwise used, lets fix this by doing

```
assert(type.getStruct().fields.size() > field);
``

and getting rid of the variable entirely.